pages/index.js

```diff
@@ -36,16 +46,8 @@ const Home = ({ data }) => {
       </button>
       <h2 className={style.smallHeading}>{monthNames[month]}</h2>
       <br />
-      <ul className={style.grid}>
-        <li className={style.card} key={data.properties.periods.startTime}>
-          <p className={style.day}>
-            {data.properties.periods[0].startTime[8]}
-            {data.properties.periods[0].startTime[9]}
-          </p>
-          <p>{data.properties.periods[0].temperature}°F</p>
-          <p>{data.properties.periods[0].name}</p>
-        </li>
-      </ul>
+      
+      <CurrentWeather data={data} />
```

Comment:
I find that variable name `data` can be hard to understand what is in the variable since the name is somewhat generic. What do you think about a specific name like `weatherData`?

Reply:
ok will change

Reply:
Generally I like to resolve all the discussions / make any changes from reviews before merging. Does that sound good to you?

Reply:
ok